capabilities/blob/replicate.go

```diff
@@ -0,0 +1,32 @@
+package blob
+
+import (
+	"github.com/ipld/go-ipld-prime/datamodel"
+	"github.com/storacha/go-ucanto/core/ipld"
+	"github.com/storacha/go-ucanto/core/schema"
+	"github.com/storacha/go-ucanto/validator"
+
+	"github.com/storacha/go-libstoracha/capabilities/types"
+)
+
+const ReplicateAbility = "blob/replicate"
```

Comment:
this should be "space/blob/replicate" per the RFC and should be in the `space/blob` folder along with the other Blob capabilities that are initially invoked by the client. the folder probably didn't exist when you started the PR but it should be moved before merge.

Reply:
same with changes to the ipld schema file

Reply:
done.